src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/failover/FailoverConnectionPlugin.java

```diff
@@ -881,16 +871,6 @@ && shouldExceptionTriggerConnectionSwitch(originalException)) {
     throw wrapperException;
   }
 
-  private String getClusterKeyword(Matcher matcher) {
```

Comment:
is this function not needed anymore?

Reply:
It's not needed anymore after updating the parsing functions